client/src/app/about/page.tsx

```diff
@@ -0,0 +1,83 @@
+import { AboutHero } from "@/ui/about/AboutHero";
+import { Typography } from "@mui/material";
+import { AboutProfile, IAboutProfile } from "@/ui/about/AboutProfile";
+import { AboutMission } from "@/ui/about/AboutMission";
+export default function Page() {
+  const aboutProfiles: IAboutProfile[] = [
+    {
+      name: "Mekhi Hart Dela Cruz",
+      role: "Lead",
+      description:
+        "Mekhi Hart is a Software Engineer at Netflix and has vast experience developing software from video games, mobile, and web. He also like to rock climb in his spare time.",
+      linkedInURl: "https://www.linkedin.com/in/mekhihart-delacruz/",
+      profilePic: "/about/profilepic_mekhi.png",
+    },
+    {
+      name: "Mark Carson",
+      role: "Full-Stack",
+      description:
+        "Mark Carson has a diverse background in full-stack development and machine learning applications. He spends time AFK playing piano and in the kitchen.",
+      linkedInURl: "https://www.linkedin.com/in/markcarsondev/",
+      profilePic: "/about/profilepic_mark.png",
+    },
+    {
+      name: "David Serrano",
+      role: "Frontend",
+      description:
+        "David Serrano is an aspiring software developer and or IT technician with a passion for fitness and a constant drive to learn and grow.",
+      linkedInURl: "https://www.linkedin.com/in/david-serrano-0a6a5523a/",
+      profilePic: "/about/profilepic_david.png",
+    },
+    {
+      name: "Kate Steer",
+      role: "Backend",
+      description:
+        "Kate Steer is a software developer with experience in database management and architecture, well as application design. In her free time, she enjoys creating music and reading.",
+      linkedInURl: "https://www.linkedin.com/in/kate-steer-4149001a6/",
+      profilePic: "/about/profilepic_kate.png",
+    },
+    {
+      name: "Howie",
```

Comment:
Hey, my name is Hao Phan, you can call me "Howie" on daily basis, "Howie" is just a nickname :(((

Reply:
oops let me change that lmao